src/components/mediaViewer/helpers/ghostAnimation.ts

```diff
@@ -1,21 +1,20 @@
-import { requestMutation } from '../../../lib/fasterdom/fasterdom';
-
 import type { ApiDimensions, ApiMessage } from '../../../api/types';
 import { MediaViewerOrigin } from '../../../types';
 
 import { ANIMATION_END_DELAY, MESSAGE_CONTENT_SELECTOR } from '../../../config';
+import { requestMutation } from '../../../lib/fasterdom/fasterdom';
```

Comment:
@DmitriyWebDev no changes in that file except organized imports, it's not part of what this PR is trying to solve, from my POV better to undo the changes and leave only relevant part.